sinopia_client/docs/LDPApi.md

```diff
@@ -116,6 +118,7 @@ var resource = new SinopiaServer.Resource(); // Resource | Resource to insert in
 
 var opts = { 
   'slug': "slug_example", // String | The suggested URI path for the resource.
+  'link': "link_example", // String | specifies container type.
```

Comment:
Would it be helpful to spell this out? Like...

`"<http://www.w3.org/ns/ldp#RDFSource>; rel=\"type\"" ...`

Assuming that we are using LDP-RS for Sinopia resources, of course.

Reply:
I like the idea of specific examples here in correct syntax.

Reply:
similar comment about this being auto-gen and me being hesitant to update it